_i18n/en.yml

```diff
@@ -0,0 +1,81 @@
+---
+global:
+  brazilian portuguese: Português
+  english: English
+header:
+  stories: Stories
+  explore: Explore
+  about: About
+support:
+  fund_us: Support the project
+home:
+  about: Artificial Intelligence for social control of public administration
+  about_rosie: Operação Serenata de Amor created a Robot - An artificial intelligence capable of analysing each request for reimbursement from congresspeople and identify it’s illegal probability.
+  last_histories: Last published histories
+  loading: Loading...
+  more_posts: More posts
```

Comment:
What about _More stories_? (to keep the coherence with the other line that mention posts as stories)